src/java/voldemort/VoldemortClientShell.java

```diff
@@ -117,7 +117,7 @@ public VoldemortClientShell(ClientConfig clientConfig,
         try {
             factory = new SocketStoreClientFactory(clientConfig);
             client = factory.getStoreClient(storeName);
-            adminClient = new AdminClient(clientConfig);
+            adminClient = new AdminClient(clientConfig.setIdentifierString("admin"));
```

Comment:
Sounds good. Might be worth amending that commit comment to mention this. It's not that important though. As you wish.